pkg/observability/observability.go

```diff
@@ -1,6 +1,12 @@
 // Package observability provides tools for working with open census.
 package observability
 
+import "go.opencensus.io/tag"
+
 const (
 	MetricRoot = "en-verification-server"
 )
+
+var (
+	RealmTagKey, _ = tag.NewKey("realm")
```

Comment:
This should probably be inside an `init()` so we can do something with the error? I don't enjoy that the error just swallows - seems hard to debug.

Reply:
done - the only error that throws is if you use invalid characters in the tag or it's too long, but there is no risk unless someone changes the tag.